packages/patrol/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## Unreleased
+
+- No throw error in `selectFineLocation` when it's already selected. (#2302)
+
```

Comment:
`3.11.0` is actually unreleased. Can you fix it to make both those changes (yours and #2312) under `## Unreleased`?

Reply:
done